src/platform/mod.rs

```diff
@@ -22,9 +23,10 @@ use crate::{device, logger};
 pub type Plat = select_env!["MIRALIS_PLATFORM_NAME":
     "miralis"     => miralis::MiralisPlatform
     "visionfive2" => visionfive2::VisionFive2Platform
+    "premierp550"             => premierp550::PremierP550Platform
```

Comment:
Minor nit: align the '=>' properly. Is it a tab on your machine displayed with a different width on your machine?